sage-ios/SAGE/SAGE/CheckinRequestTableViewCell.swift

```diff
@@ -18,12 +18,14 @@ class CheckinRequestTableViewCell: UITableViewCell {
     var checkButton = UIButton()
     var xButton = UIButton()
     
+    var checkinID: Int?
```

Comment:
hmm.... this is not too good, but i see where ur coming from. if i had to do this id probably just do this too lolol. fuck, do you have any suggestions?